dbt_project.yml

```diff
@@ -2,7 +2,7 @@ name: 'xdb'
 config-version: 2
 version: '0.0.1'
 macro-paths: ["macros"]
-clean-targets: ["target","dbt_modules"]
-require-dbt-version: ">=0.14.0"
+clean-targets: ["target","dbt_packages"]
+require-dbt-version: ">=1.0.0"
```

Comment:
Do we use xdb in any of the old repos that are using older dbt versions? This change will force anything using xdb to be at least 1.0, which I think is fine. Just need to verify that it doesn't break anything for us.

Reply:
I guess only you can know what repositories might use xdb

Reply:
True. I don't think it will. The deprecated data models repo doesn't seem to use it at all, so we should be fine on that front.